src/development/platform-integration/android/restore-state-android.md

```diff
@@ -0,0 +1,185 @@
+--- 
+title: "State restoration for Android apps"
+description: "How to restore the state of your Android app after it's been killed by the OS."
+---
+
+When a user runs a mobile app and then selects another
+app to run, the first app is moved to the background,
+or _backgrounded_. The operating system (both iOS and Android)
+typically kill the backgrounded app to release memory and
```

Comment:
nit: I would replace "typically" with "may" to be a little softer. It depends on a lot of circumstances whether you get killed or not..